apps/dotcom/client/src/tla/app/zero-polyfill.ts

```diff
@@ -179,14 +179,7 @@ export class Zero {
 				if (store?.files.find((f) => f.id === data.id)) {
 					throw new Error('file already exists')
 				}
-				this.makeOptimistic([
-					{ table: 'file', event: 'insert', row: data },
-					{
-						table: 'file_state',
-						event: 'insert',
-						row: { fileId: data.id, userId: store.user.id, firstVisitAt: Date.now() } as any,
-					},
-				])
```

Comment:
remember we had that "cannot update file state for another user: undefined" bug in sentry? this is why. I hoisted this out into the TldrawApp.createFile method and made sure it supplies all the right properties